<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>434</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>即时通讯客户端</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="styleSheet">
       <string>color: red; font-weight: bold;</string>
      </property>
      <property name="text">
       <string>未登录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="contactsPage">
       <layout class="QVBoxLayout" name="verticalLayout_contacts">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="contactsTab">
           <attribute name="title">
            <string>联系人</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="addContactButton">
              <property name="text">
               <string>添加联系人</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="contactsListWidget"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="groupsTab">
           <attribute name="title">
            <string>群组</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="createGroupButton">
              <property name="text">
               <string>创建群组</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addGroupMemberButton">
              <property name="text">
               <string>添加群成员</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="groupsListWidget"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="chatPage">
       <layout class="QVBoxLayout" name="verticalLayout_chat">
        <item>
         <widget class="QPushButton" name="backButton">
          <property name="text">
           <string>返回</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="chatContainer" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>100</width>
            <height>30</height>
           </rect>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="refreshButton">
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="logoutButton">
        <property name="text">
         <string>退出登录</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>350</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
